File: app/src/main/java/agi/erecreditsmanager/AddedLecture/AddedLectureLayout.java
package agi.erecreditsmanager.AddedLecture;

import android.content.Context;
import android.view.LayoutInflater;
import android.widget.Button;
import android.widget.CheckedTextView;
import android.widget.RelativeLayout;

import agi.erecreditsmanager.R;

public class AddedLectureLayout extends RelativeLayout {

    Context context;
    CheckedTextView addedLectureCreditTextView;
    Button deleteButton;

    public AddedLectureLayout(Context context) {
        super(context);
        this.context = context;

        init();
    }

    private void init() {
        LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
        inflater.inflate(R.layout.addedlecture, this, true);

        addedLectureCreditTextView = (CheckedTextView) findViewById(R.id.addedLectureCreditTextView);
        deleteButton = (Button) findViewById(R.id.deleteButton);
    }

    public void setAddedLectureCreditTextView(String text) {
        addedLectureCreditTextView.setText(text);
    }

    public CheckedTextView getAddedLectureCreditTextView() {
        return addedLectureCreditTextView;
    }

    public Button getDeleteButton() {
        return deleteButton;
    }
}

File: app/src/main/java/agi/erecreditsmanager/AddedLecture/AddedLectureLayout.kt
package agi.erecreditsmanager.AddedLecture

import agi.erecreditsmanager.R
import android.content.Context
import android.view.LayoutInflater
import android.widget.Button
import android.widget.CheckedTextView
import android.widget.RelativeLayout
import kotlinx.android.synthetic.main.addedlecture.view.*

class AddedLectureLayout(context : Context) : RelativeLayout(context) {
    init {
        LayoutInflater.from(context).run {
            inflate(R.layout.addedlecture, this@AddedLectureLayout, true)

            addedLectureCreditTextView
            deleteButton
        }
    }

    //MainAdapter 재구현 후 삭제
    fun setAddedLectureCreditTextView(text : String) {
        addedLectureCreditTextView.text = text
    }

    fun getAddedLectureCreditTextView() : CheckedTextView {
        return addedLectureCreditTextView
    }

    fun getDeleteButton() : Button {
        return deleteButton
    }
}